Assets/MirageXR/Player/Resources/Prefabs/Calibration/CalibrationGuide.prefab

```diff
@@ -78,7 +78,7 @@ MonoBehaviour:
     m_HorizontalOverflow: 1
     m_VerticalOverflow: 1
     m_LineSpacing: 1
-  m_Text: OK
```

Comment:
I'm not sure if `ok` should be changed to `close`

Reply:
Hi @cawke This is a regression problem. At some point this text was added and then, for some reason, returned to the 'OK' value. (I understand the test for 'OK' is 'Close' for IOS, Android and Windows). So using TDD principles, I added a test for this.